### ISSUE TITLE
Error loading preference bundle for tweak.

### ISSUE THREAD

**Issue by thecolbster91** [pwn20wndstuff/Undecimus#2054]
**Describe the bug**
What’s up? I purchased two tweaks this week 
callled Dope Convos and Genesis. When I went so settings, I get this error for both of these tweaks. What can I do to solve these errors?

**To Reproduce**
1. Go to settings
2. Click on Tweaks
3. Scroll down to DopeConvos

**Screenshots
![524E006C-3EC4-42F2-A850-D2401E1262A1](https://user-images.githubusercontent.com/35181281/83980570-e79c2000-a8e4-11ea-8660-b10fe3e8ce05.png)

![14BA7AC7-0EE2-452D-B4A6-E0FF4CC944C3](https://user-images.githubusercontent.com/35181281/83980571-e965e380-a8e4-11ea-8755-f2af1a6113e6.png)

**Device (please complete the following information):**
 - iOS Version: 13.5
 - iOS Device: iPhone XS Max
 - unc0ver Version: 5.0.0

**Place an "x" between the brackets if true:**
 - [x] this is a bug others will be able to reproduce
 - [ ] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ ] this issue is present after a rootfs restore
 - [x] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.

**Comment by thekio1984** [comment]
I also have tweaks that should work, but dont, on my SE2, yet still work on my old SE both on 13.5, even with all usual supporting tweaks installed (rocketbootstrap, applist, preference loader) some tweaks just refuse to work with u0.. at guess its either install bracketed tweaks.. if there already installed, devs need to update there tweaks. Unless (this is a long shot & i also doubt) its an issue with substitute & it's compatibility with substrate made tweaks (if thats even a thing lol) 🤷🏼‍♂️

**Comment by anshulkatare** [comment]
Facing the same with scorpion, airport, Lynx and other tweaks as well.

**Comment by liaocm** [comment]
This seems to be an A12+ issue. The (or... one?) cause of it is described in #2045 . You'll need to wait for the developer to update and recompile the tweak.

**Comment by Hamlock-maneuverr** [comment]
What preferenceloader are you using?

**Comment by Vither-05** [comment]
Yes I'm using iPad 8 A12 and my iPad OS version is 14.2. Using Unc0ver 6.1.0 to jailbreak and when installing some tweaks its still not work. Still no solution?
